source/user-manual/capabilities/file-integrity/compatibility-matrix.rst

```diff
@@ -0,0 +1,28 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+  :description: Learn more about File Integrity Monitoring, a key capability of Wazuh. Check out how it works and some practical use cases. 
```

Comment:
Please let's update the metadata description explaining what is the content of this page.

Reply:
Updated